master/pkg/model/test_utils.go

```diff
@@ -37,7 +37,7 @@ func ExperimentModel(opts ...ExperimentModelOption) *Experiment {
 				RawMaxLength: &maxLength,
 			},
 		},
-		RawEntrypoint: &expconf.Entrypoint{RawEntrypoint: "model_def:SomeTrialClass"},
+		RawEntrypoint:      &expconf.Entrypoint{RawEntrypoint: "model_def:SomeTrialClass"},
```

Comment:
I don't know if this is picked up by lint, but I think you can remove this change and keep the file unchanged

Reply:
I feel like this was caused by lint as I didn't make it.

Reply:
this is probably lint not getting run with tags properly or something. i'll fix.